addons/Popochiu/Engine/Objects/Room/PopochiuRoom.gd

```diff
@@ -272,9 +289,20 @@ func _move_along_path(distance):
 			# Play the animation on the character
 			_moving_character.play_walk(_path[0])
 			# Move the character on the destination
-			_moving_character.position = last_point.linear_interpolate(
-				_path[0], distance / distance_between_points
-			)
+			#in order to run anti-glide animation you need to add call method 
+			#track in animation player and call update_position every frame
+			#additional condition to avoid crash if update_position method
+			#would be deleted
+			if (_moving_character.anti_glide_animation == true 
+			and _moving_character.has_method("update_position")
+			):
+				moving_character_position_stored = last_point.linear_interpolate(
+					_path[0], distance / distance_between_points
+				)
+			else:
+				_moving_character.position = last_point.linear_interpolate(
+					_path[0], distance / distance_between_points
+				)
```

Comment:
Since it's the same math, it could be done before the if statement to simplify the statement
```suggestion
			var next_position := last_point.linear_interpolate(
					_path[0], distance / distance_between_points
			)
			if (_moving_character.anti_glide_animation
			  and _moving_character.has_method("update_position")
			):
				moving_character_position_stored = next_position
			else:
				_moving_character.position = next_position
```

Reply:
Looks like the indentation on my suggestion is weird, I would suggest not applying this change directly but using it as a guideline.

Reply:
@edmundito @mapedorr thanks for your comments. I will apply them together with changing moving_character_position_stored  into making PopochiuCharacter field (to avoid interference between multiple moving characters)